tests/installation/bootloader_s390.pm

```diff
@@ -330,6 +335,7 @@ sub run {
     # activate console so we can call wait_serial later
     # skip activate serial console since 11sp4 has issue bsc#1159521: iucvconn command not exist
     if (!is_sle('=11-sp4')) {
+        #sleep(2000);
```

Comment:
let's drop this.

Reply:
1825d85eea6b903a2c986748234f860179c5f973